techniques/css/C42.html

```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml" lang="en">
 	<head>
-		<title>Using min-height and min-width to ensure sufficient target spacing</title>
+		<title>Using CSS to ensure sufficient target spacing</title>
```

Comment:
why not mirroring the h1 hwere?

Reply:
Not sure why the `<title>` is showing like that here—I caught that and fixed it—[check out the diff](https://github.com/w3c/wcag/pull/3231/files#diff-afebdf00ca2405f3d441810e385c10860d625945f67cdabe5c636d26ceeb5a90).